.github/workflows/test_pr_creation.yml

```diff
@@ -0,0 +1,65 @@
+name: Prepare Jib release
+on:
+  workflow_dispatch:
+    inputs:
+      project:
+        description: Jib project to release
+        required: true
+        default: (build-plan | core | maven | gradle | extension-common | maven-extension | gradle-extension)
+      release_version:
+        description: new release version
+        required: true
+        default: (for example, 0.1.0)
+
+jobs:
+  release:
+    name: Prepare Jib release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v2
+
+      - name: Check input
+        run: |
+          echo '* input project: "${{ github.event.inputs.project }}"'
+          case ${{ github.event.inputs.project }} in
+            build-plan|core|maven|gradle|extension-common|maven-extension|gradle-extension) ;;
+            *) echo 'invalid input project name "${{ github.event.inputs.project }}"'
+               exit 1
+               ;;
+          esac
+          if [[ ! "${{ github.event.inputs.release_version }}" =~ ^[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
+            echo 'version "${{ github.event.inputs.release_version }}" not in ###.###.### format'
+            exit 1
+          fi
+      - name: Build project
+        run: |
+          # TODO: run integration test? (Requries auth with GCP.)
+          ./gradlew clean build --stacktrace
+      - name: Run Gradle release
+        run: |
+          git checkout -b ${{ github.event.inputs.project }}-release-v${{ github.event.inputs.release_version }}
+          git config user.email ${{ github.actor }}@users.noreply.github.com
+          git config user.name ${{ github.actor }}
+          PROJECT=$( case ${{ github.event.inputs.project }} in
+            extension-common) echo jib-plugins-extension-common ;;
+            maven-extension)  echo jib-maven-plugin-extension-api ;;
+            gradle-extension) echo jib-gradle-plugin-extension-api ;;
+            maven|gradle)     echo jib-${{ github.event.inputs.project }}-plugin ;;
+            *)                echo jib-${{ github.event.inputs.project }} ;;
+          esac )
+          # This creates the tag (e.g., "v0.1.0-gradle") and pushes the updated
+          # branch (e.g., "gradle-release-v0.1.0") and the new tag.
+          ./gradlew "${PROJECT}":release \
+            -Prelease.useAutomaticVersion=true \
+            -Prelease.releaseVersion=${{ github.event.inputs.release_version }}
+      - name: Create pull request
+        uses: repo-sync/pull-request@v2
+        id: create-pr
+        with:
+          # Provided by Actions. No need to create a token.
+          secret: ${{ secrets.GA_SECRET }}
+          source_branch: ${{ github.event.inputs.project }}-release-v${{ github.event.inputs.release_version }}
```

Comment:
I think then this can be any existing branch for testing, e.g., `fix-2195-tar-npe`.